src/test/scala/firrtlTests/ReplSeqMemTests.scala

```diff
@@ -45,6 +45,22 @@ class ReplSeqMemSpec extends SimpleTransformSpec {
     )
   }
 
+  def checkGenMemVerilog(input: String, mems: Set[MemConf], additionalAnnos: Annotation*): Unit = {
+    Seq(true, false).foreach { genBlackBox =>
+      val annos = Seq(GenVerilogMemBehaviorModelAnno(genBlackBox = genBlackBox)) ++ additionalAnnos
+      val res = compileAndEmit(CircuitState(parse(input), ChirrtlForm, annos))
+      // Check correctness of firrtl
+      parse(res.getEmittedCircuit.value)
+      // Check the emitted Verilog
+      mems.foreach { mem =>
+        val file = new java.io.File(mem.name + ".v")
+        require(file.exists(), s"${file.getName} should be emitted!")
+        // todo check content (using verilator?)
+        file.delete()
```

Comment:
compiling by verilator to check verilog is correct.

Reply:
Is `--lint-only` enough? I see they using it in `lintVerilog`. If it is enough, then [`e0859e8` (#2202)](https://github.com/chipsalliance/firrtl/pull/2202/commits/e0859e84b3f507f47390b5b6838cb1b3fab45cdb) fixed this.